SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -345,15 +210,20 @@ void InterestFactory::AddPlayerControllerActorInterest(Interest& OutInterest, co
 {
 	QueryConstraint LevelConstraint = CreateLevelConstraints(InActor);
 
-	AddSystemQuery(OutInterest, InActor, InInfo, LevelConstraint);
+	// Always relevant and always interested
+	AddAlwaysSomethingQuery(OutInterest, InActor, InInfo, LevelConstraint);
+	bool ShouldAddUserQueries = HasUserDefinedConstraint(InActor);
```

Comment:
```suggestion
	bool bShouldAddUserQueries = HasUserDefinedConstraint(InActor);
```

Reply:
etc.

Reply:
done 'n' moved around a bit 'n' also renamed the function